### PR TITLE
[SPARK-26073][SQL][FOLLOW-UP] remove invalid comment as we don't use it anymore

### PR DESCRIPTION
**Pull request by heary-cao**
## What changes were proposed in this pull request?

remove invalid comment as we don't use it anymore
More details: https://github.com/apache/spark/pull/22976#discussion_r233764857

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by heary-cao** [comment]
cc @cloud-fan, @viirya

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala [hunk 1]
```diff
@@ -76,8 +76,6 @@ object GenerateOrdering extends CodeGenerator[Seq[SortOrder], Ordering[InternalR
       row: String,
       ordering: Seq[SortOrder]): Seq[ExprCode] = {
     ctx.INPUT_ROW = row
-    // to use INPUT_ROW we must make sure currentVars is null
-    ctx.currentVars = null
```
Comment (cloud-fan) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala]: I think the conclusion was to keep it?

**Comment by cloud-fan** [comment]
ok to test

**Comment by cloud-fan** [comment]
add to whitelist

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala (hunk 1 above)
Comment (viirya) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala]: I think so.

**Comment by kiszk** [comment]
LGTM, pending Jenkins

**Review by viirya — APPROVED** [review]

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Comment by heary-cao** [comment]
thanks, @cloud-fan, @viirya, @kiszk.

**Pull request closed without merging**